### ISSUE TITLE
Build failure: nvimpager

### ISSUE THREAD

**Issue by Andy3153** [NixOS/nixpkgs#313966]
### Steps To Reproduce

1. build nvimpager

### Build log

```console
$ doas nixos-rebuild switch

error: builder for '/nix/store/lq2izg5wi60n2dy08y4w8y43i69fbzfp-nvimpager-0.12.0.drv' failed with exit code 2;
       last 10 log lines:
       > ok 133 - lua functions check_escape_sequences only checks files with filetype 'text'
       > ok 134 - lua functions check_escape_sequences finds ansi escape sequences
       > ok 135 - lua functions detect_parent_process detects man
       > ok 136 - lua functions detect_parent_process detects pydoc
       > ok 137 - lua functions detect_parent_process detects ruby
       > ok 138 - lua functions detect_parent_process detects perl
       > ok 139 - lua functions detect_parent_process detects git
       > ok 140 - lua functions detect_parent_process returns nil for unknown parents
       > 1..140
       > make: *** [makefile:38: test] Error 1
       For full logs, run 'nix log /nix/store/lq2izg5wi60n2dy08y4w8y43i69fbzfp-nvimpager-0.12.0.drv'.
error: 1 dependencies of derivation '/nix/store/q2wfs75gnk8hz7dlg0w7nmn7a9k37hfg-system-path.drv' failed to build
error: 1 dependencies of derivation '/nix/store/n6p56f2ya0n87z2lgnbjs1nm92zb1s08-nixos-system-sparkle-24.05.20240521.5710852.drv' failed to build
```

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.9.1-zen1, NixOS, 24.05 (Uakari), 24.05.20240521.5710852`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by eclairevoyant** [comment]
This was already reported in #313134 and fixed by #313359

**Comment by Andy3153** [comment]
Sorry for bothering. Searching didn't bring any results to me
